inc/ui/namespace.php

```diff
@@ -4,36 +4,36 @@
-	</g>
-</svg>
-END;
+use WP_Admin_Bar;
 
 function bootstrap() {
 	add_action( 'admin_menu', __NAMESPACE__ . '\\register_menu' );
+	add_action( 'admin_bar_menu', __NAMESPACE__ . '\\admin_bar_menu', 11 );
 }
 
 function register_menu() {
-	$hook = add_menu_page(
-		'Human Made Platform',
-		'Platform',
+	$hook = add_submenu_page(
+		null,
+		'Documentation',
+		'Documentation',
 		'edit_posts',
-		'hm-platform',
-		__NAMESPACE__ . '\\render_page',
-		'data:image/svg+xml;base64,' . base64_encode( LOGO ),
-		2
+		'hm-platform-documentation',
+		__NAMESPACE__ . '\\render_page'
 	);
 
 	add_action( sprintf( 'load-%s', $hook ), __NAMESPACE__ . '\\load_page' );
 }
 
+function admin_bar_menu( WP_Admin_Bar $wp_admin_bar ) {
+	// Add WordPress.org link
+	$wp_admin_bar->add_menu( [
+		'parent'    => 'hm-platform-logo',
```

Comment:
Needs updating for latest changes in https://github.com/humanmade/platform-cms/pull/22